src/App.tsx

```diff
@@ -1,13 +1,94 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import React, { useEffect, useRef, useState } from 'react';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodosList } from './components/TodosList';
+import { FilterType } from './types/FilterType';
+import { Footer } from './components/Footer';
+import classNames from 'classnames';
+
+enum ErrorType {
+  EmptyTitle,
+  UnableToLoad,
+  UnableToDelete,
+  UnableToUpdate,
+}
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const inputValue = useRef<HTMLInputElement>(null);
+  const [error, setError] = useState<ErrorType | null>(null);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>([]);
+  const [query, setQuery] = useState<FilterType>(FilterType.All);
+
+  const downloadTodos = async () => {
+    try {
+      const tmp = await getTodos();
+
+      setTodos(tmp);
+      setFilteredTodos(tmp);
+    } catch {
+      setError(ErrorType.UnableToLoad);
+    }
+  };
+
+  useEffect(() => {
+    downloadTodos();
+  }, []);
+
+  useEffect(() => {
+    if (todos) {
+      switch (query) {
+        case FilterType.Active:
+          setFilteredTodos(todos.filter(todo => todo.completed === false));
+          break;
+        case FilterType.Complited:
+          setFilteredTodos(todos.filter(todo => todo.completed === true));
```

Comment:
you can shorten it to 
```suggestion
          setFilteredTodos(todos.filter(todo => !todo.completed));
          break;
        case FilterType.Complited:
          setFilteredTodos(todos.filter(todo => todo.completed));
```